packages/epo-widget-lib/src/charts/ClippingContainer/index.tsx

```diff
@@ -0,0 +1,31 @@
+import { FunctionComponent, PropsWithChildren, useId } from "react";
+
+export interface ClippingContainerProps {
+  x: number;
+  y: number;
+  width: number;
+  height: number;
+  className?: string;
+}
+
+const ClippingContainer: FunctionComponent<
```

Comment:
this container is useful for clipping content that goes beyond the current domain, for large datasets we should definitely be taking a subset of the data but in other cases it may be useful to render the whole set and just hide what's outside the domain